src/Tribe/Ticket_Object.php

```diff
@@ -511,16 +511,29 @@ public function original_stock() {
 		 * @return boolean
 		 */
 		public function is_in_stock() {
-			// if we aren't tracking stock, then always assume it is in stock
+			// if we aren't tracking stock, then always assume it is in stock.
 			if ( ! $this->managing_stock() ) {
 				return true;
 			}
 
-			$remaining = $this->inventory();
+			/* @var Tribe__Cache $cache */
```

Comment:
:warning: This comment is 50% valid code; is this commented out code?

<sub>hash: 1920a00c103a3d9d4340fb1a347edccd</sub>